django/src/rdwatch_scoring/views/vector_tile.py

```diff
@@ -0,0 +1,234 @@
+    cache_key = _get_vector_tile_cache_key(
+        evaluation_run_uuid, z, x, y, latest_timestamp
+    )
+
+    tile = cache.get(cache_key)
+
+    # Generate the vector tiles and cache them if there's no hit
+    if tile is None:
+        envelope = Func(z, x, y, function='ST_TileEnvelope')
+        intersects = Q(
+            Func(
+                'transformedgeom',
+                envelope,
+                function='ST_Intersects',
+                output_field=BooleanField(),
+            )
+        )
+        transform = Func(
+            'geomfromtext', 3857, function='ST_Transform', output_field=GeometryField()
+        )
+        mvtgeom = Func(
+            'transformedgeom',
+            envelope,
+            function='ST_AsMVTGeom',
+            output_field=Field(),
+        )
+        geomfromuniongeometrytext = Func(
+            'union_geometry', 4326, function='ST_GeomFromText', output_field=Field()
+        )
+        geomfromobservationtext = Func(
+            'geometry', 4326, function='ST_GeomFromText', output_field=Field()
+        )
+
+        site_queryset = (
+            Site.objects.filter(evaluation_run_uuid=evaluation_run_uuid)
+            .alias(geomfromtext=geomfromuniongeometrytext)
+            .alias(transformedgeom=transform)
+            .filter(intersects)
+            .values()
+            .annotate(
+                id=F('site_id'),
+                mvtgeom=mvtgeom,
+                configuration_id=F('evaluation_run_uuid'),
+                label=Case(
+                    When(
+                        Q(status_annotated__isnull=False),
+                        Lower(Replace('status_annotated', Value(' '), Value('_'))),
+                    ),
+                    default=Value('unknown'),
+                ),  # This needs a version to be scoring coloring,
+                # but that needs some coordination with kitware
+                timestamp=ExtractEpoch('evaluation_run_uuid__start_datetime'),
+                timemin=ExtractEpoch('start_date'),
+                timemax=ExtractEpoch('end_date'),
+                performer_id=F('originator'),
+                region=F('region_id'),
+                groundtruth=Case(
+                    When(
+                        Q(originator='te'),
+                        True,
+                    ),
+                    default=False,
+                ),
+                site_polygon=Value(False, output_field=BooleanField()),
+            )
+        )
+        (
+            site_sql,
+            site_params,
+        ) = site_queryset.query.sql_with_params()
+
+        observations_queryset = (
+            Observation.objects.filter(
+                site_uuid__evaluation_run_uuid=evaluation_run_uuid
+            )
+            .alias(geomfromtext=geomfromobservationtext)
+            .alias(transformedgeom=transform)
+            .filter(intersects)
+            .values()
+            .annotate(
+                id=F('uuid'),
```

Comment:
```suggestion
                id=F('uuid'),
                timestamp=F('date'),
```

So we in our version of the Observation model have a value called timestamp.  This is returned and provides the timestamp on hover.  That's why it isn't available on this system.  They have the 'date' property.  We just need to rename it to timestamp for it to show up.

![image](https://github.com/ResonantGeoData/RD-WATCH/assets/61746913/f583cfa1-9115-4e95-9b56-3bbd4ec5b67f)